cli/src/semgrep/app/scans.py

```diff
@@ -228,21 +269,21 @@ def report_findings(
             all_matches,
             lambda match: bool(match.is_ignored),
         )
-
+        findings = [
+            match.to_app_finding_format(commit_date).to_json() for match in new_matches
+        ]
+        ignores = [
+            match.to_app_finding_format(commit_date).to_json() for match in new_ignored
+        ]
         findings_and_ignores = {
             # send a backup token in case the app is not available
             "token": os.getenv("GITHUB_TOKEN"),
             "gitlab_token": os.getenv("GITLAB_TOKEN"),
-            "findings": [
-                match.to_app_finding_format(commit_date).to_json()
-                for match in new_matches
-            ],
+            "findings": findings,
             "searched_paths": [str(t) for t in targets],
-            "rule_ids": all_ids,
-            "ignores": [
-                match.to_app_finding_format(commit_date).to_json()
-                for match in new_ignored
-            ],
+            "rule_ids": rule_ids,
+            "cai_ids": cai_ids,
```

Comment:
Do we need to send back cai_ids?

Reply:
I think we might, since their app pre-dates us getting rid of CAI and I don't know if they have CAI enabled. Without this.